### DISCUSSION TITLE
External dashboard app - resource pickers.

### DISCUSSION THREAD

**Discussion by gufio** [mirumee/saleor-dashboard#3726] [General]
Hello. I'm developing an external dashboard app and here is my problem. Let's say have a products list view and we want to do some stuff in external app ("POPUP" target) based on products selection. Is there a way of somehow pass selected products into app and maybe filters other way than in url? I want this to work similar to "export products" and I don't want to create custom product list in my app.

There's an issue #3605 for products resource picker. How is this going to work? Will it be something like shopify resource pickers?

**Comment by timuric** [comment]
Hey @gufio thanks for raising this. We are working on resource picker and will expose API to apps, stay tuned for updates
